pywbem_mock/_wbemconnection_mock.py

```diff
@@ -480,6 +481,74 @@ def compile_mof_string(self, mof_str, namespace=None, search_paths=None,
 
         self._merge_repos(mof_repo)
 
+    def compile_dmtf_schema(self, schema_version, schema_root_dir, class_names,
+                            use_experimental=False, verbose=False):
+        """
+        Compile the classes defined by `classes_names` and their dependent
+        classes from the DMTF CIM schema version defined by
+        'schema_version' and keep the downloaded DMTF CIM schema in the
+        directory defined by `schema_dir`.
+
+        This method uses :class:`~pywbem_mock.DMTFCIMSchema` to get the schema
+        from the DMTF, save it in the `schema_dir` directory, expand it and make
+        a list of leaf classes from this directory that are to be compiled.
+
+        It automatically compiles all of the DMTF qualifier declarations that
+        are in the files `qualifiers.mof` and `qualifiers_optional.mof`.
+
+        Parameters:
+
+          schema_version (tuple of 3 integers (m, n, u):
+            Represents the DMTF CIM schema version where:
+
+            * m is the DMTF CIM schema major version
+            * n is the DMTF CIM schema minor version
+            * u is the DMTF CIM schema update version
+
+            This must represent a DMTF CIM schema that is available from the
+            DMTF web site.
+
+          schema_dir (:term:`string`):
+            Directory into which the DMTF CIM schema is installed or will be
+            installed.  A single `schema_dir` can be used for multiple
+            schema versions because subdirectories are uniquely defined by
+            schema version and schema_type (i.e. Final or Experimental).
+
+          classes_names (:term:`py:list` of :term:`string`):
```

Comment:
Incorrect parameter name, change to "class_names"

Reply:
DONE. Found one other of same issue